### DISCUSSION TITLE
I tried to embed YouTube videos but it doesn't work

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#568] [Q&A] [answered]
It shows me a white space with "www.youtube.com refused to connect."

Is this behavior expected because of MDX?

**Comment by fuma-nama** [comment]
Probably problem related to YouTube or the iframe, I don't think Fumadocs MDX caused the problem

**Reply by Ali-Hussein-dev** [reply, marked as answer]
The src of iframe is not the same as the URL to a YT video, it must be copied from the iframe embed
